sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql

```diff
@@ -32,8 +32,8 @@ WHERE
   normalized_app_name != "Firefox Desktop"
 UNION ALL
 SELECT
-  submission_date AS activity_date,
   first_seen_date AS cohort_date,
+  submission_date AS activity_date,
```

Comment:
IMO it's most intuitive to put the partitioning column(s) first, so if it were me I'd keep the original column order here and switch the column order in the first query.